Library/Homebrew/test/build_environment_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "build_environment"
+
+RSpec::Matchers.define :use_userpaths do
+  match do |actual|
+    actual.userpaths?
+  end
+end
+
+describe BuildEnvironment do
+  let(:env) { described_class.new }
```

Comment:
That's true, but there's also some semantic meaning attached to `subject` and I think consistency is desirable here — it's not just a hack you use to get nice syntax. When I glanced at these tests I assumed `env` to be some kind of helper object.